lib/pip.js

```diff
@@ -194,7 +194,7 @@ function installRequirements(targetFolder, serverless, options) {
       // Mount necessary ssh files to work with private repos
       dockerCmd.push(
         '-v',
-        `${process.env.HOME}/.ssh/id_rsa:/root/.ssh/id_rsa:z`,
+        `${process.env.HOME}/.ssh/${options.keyFile || 'id_rsa'}:/root/.ssh/id_rsa:z`,
```

Comment:
I think it would be more flexible if keyFile was a full path to a file:
```suggestion
        options.keyFile ? `${options.keyFile}:/root/.ssh/id_rsa:z`: `${process.env.HOME}/.ssh/id_rsa:/root/.ssh/id_rsa:z`,
```

Reply:
Or maybe search on absolute path. than current path, than .ssh.
Whay do you think?
Too much complex?

Reply:
if you want to implement what you suggested(with test coverage :wink:) I'm cool with that. Definitely more complex tho.